docs/config/dev.toml

```diff
@@ -20,11 +20,30 @@ ip_rate_limit = { type = 'ip', timeframe = 20000 }
 sid_rate_limit = { type = 'sid', timeframe = 20000 }
 
 ethereum_core_address = '0x333420fc6a897356e69b62417cd17ff012177d2b'
+# TODO: Replace with real contract address
+outpace_address = '0x333420fc6a897356e69b62417cd17ff012177d2b'
+# TODO: Replace with real contract address
```

Comment:
to make tests reproducible we can create a ganache snapshot of the required deployments and use it on startup

Reply:
This is a cool idea for enhancement!
How can we do that?